roles/dd-nodepool/files/nodepool_server.py

```diff
@@ -20,17 +20,24 @@
 class NodepoolCheck(AgentCheck):
 
     SERVICE_CHECK_NAME = 'nodepool_server.got_status'
+    IMAGES_CHECK_NAME = 'nodepool_server.got_images'
 
     def check(self, instance):
         if 'status_url' not in instance:
-            self.log.info("Skipping instance, no url found.")
+            self.log.info("Skipping instance, no status url found.")
+            return
+
+        if 'images_url' not in instance:
+            self.log.info("Skipping instance, no images url found.")
             return
```

Comment:
we probably don't want to return from both of these. If status_url isn't found then skip the status part, if images_url isn't found skip the images part.